### PR TITLE
V1.0.3

### PR DESCRIPTION
**Pull request by philschmid**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/THIRD-PARTY-LICENSES
Comment (philschmid) [inline huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/THIRD-PARTY-LICENSES:1]: Shouldn't that file be at https://publish.djl.ai/dlc-licenses/huggingface/tgi-1.0.3/THIRD-PARTY-LICENSES

Comment (haixiw) [reply huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/THIRD-PARTY-LICENSES:1]: I don't have the access to this link. BTW Eric made the change here: https://github.com/awslabs/llm-hosting-container/commit/40df5f98ac3469c4ea0aa10cbdc1694cf4f1af21 Probably we need to do the similar thing here. I could confirm it with certain team on our side.

File: huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+RUN HOME_DIR=/root && \
+    pip install requests && \
+    curl -o ${HOME_DIR}/oss_compliance.zip https://aws-dlinfra-utilities.s3.amazonaws.com/oss_compliance.zip && \
+    unzip ${HOME_DIR}/oss_compliance.zip -d ${HOME_DIR}/ && \
+    cp ${HOME_DIR}/oss_compliance/test/testOSSCompliance /usr/local/bin/testOSSCompliance && \
+    chmod +x /usr/local/bin/testOSSCompliance && \
+    chmod +x ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh && \
+    ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh ${HOME_DIR} python && \
+    rm -rf ${HOME_DIR}/oss_compliance*
+RUN curl -o /root/THIRD-PARTY-LICENSES https://publish.djl.ai/dlc-licenses/huggingface/tgi-1.0.2/THIRD-PARTY-LICENSES
```
Comment (haixiw) [inline huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu:223]: We need to update the version to 1.0.3 here as well

**Review by haixiw — CHANGES_REQUESTED** [review]

File: huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu (hunk 1 above)
Comment (philschmid) [inline huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu:223]: if we add the file int the repo we need to change this similar to https://github.com/awslabs/llm-hosting-container/commit/40df5f98ac3469c4ea0aa10cbdc1694cf4f1af21

File: huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/THIRD-PARTY-LICENSES
Comment (philschmid) [reply huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/THIRD-PARTY-LICENSES:1]: Can you make the change?

File: huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu (hunk 1 above)
Comment (philschmid) [reply huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu:223]: Yes can you make the change? or should i

Comment (philschmid) [reply huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu]: Made the thange

**Review by haixiw — APPROVED** [review]

**Pull request merged**